### DISCUSSION TITLE
Account access restriction

### DISCUSSION THREAD

**Discussion by ocalvo** [photoprism/photoprism#1253] [Q&A] [answered]
I have the need to clearly separate photos according to user access to the file.
Currently I deal with this by having photos separated into 2 separate folders, and users only have access via ACL and share rights.

Do I need to have multiple separate instances of photoprsim running? or is there a concept of "access right" in photoprism?

Thanks,

Oscar

**Comment by lastzero** [comment, marked as answer]
Yes, you need to run two instances as you need two distinct indexes for this. Otherwise, once instance would need to switch or dynamically merge indexes based on user permissions - which is complex and bug prone.
